From d9fb4fea310a7124babd8b0eed58231a2df7692c Mon Sep 17 00:00:00 2001
From: Danny Baumann <dannybaumann@web.de>
Date: Wed, 12 Sep 2018 14:01:38 +0200
Subject: [PATCH] Messaging: Fix crash of blocked participant list activity

This activity was asserting that *only* the blocked participant list
fragment is attached to it, but the lifecycle libraries also inject a
fragment for their internal purposes.

Change-Id: Iafd798337a38fc83cf45174b20a8b6a76bf20546
---
 .../android/messaging/ui/BlockedParticipantsActivity.java  | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/src/com/android/messaging/ui/BlockedParticipantsActivity.java b/src/com/android/messaging/ui/BlockedParticipantsActivity.java
index b740264..8d79155 100644
--- a/src/com/android/messaging/ui/BlockedParticipantsActivity.java
+++ b/src/com/android/messaging/ui/BlockedParticipantsActivity.java
@@ -16,12 +16,10 @@
 
 package com.android.messaging.ui;
 
-import android.app.Fragment;
 import android.os.Bundle;
 import android.view.MenuItem;
 
 import com.android.messaging.R;
-import com.android.messaging.util.Assert;
 
 /**
  * Show a list of currently blocked participants.
@@ -36,11 +34,6 @@ public class BlockedParticipantsActivity extends BugleActionBarActivity {
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
     }
 
-    @Override
-    public void onAttachFragment(final Fragment fragment) {
-        Assert.isTrue(fragment instanceof BlockedParticipantsFragment);
-    }
-
     @Override
     public boolean onOptionsItemSelected(final MenuItem item) {
         switch (item.getItemId()) {
-- 
2.17.1

